src/content/docs/distributed-tracing/infinite-tracing/set-trace-observer.mdx

```diff
@@ -153,16 +295,70 @@ To set up a trace observer:
-   * [Trace API](/docs/understand-dependencies/distributed-tracing/trace-api): once the trace observer is set up, you're finished and can start instrumenting your application.
+### Initial trace observer configuration for browser, mobile, and Lambda [#initial-config]
+
+Once you have a trace observer, you need to specify which data sources (services, apps, or entities) should have their data sent to the Infinite Tracing trace observer. Before you select your data sources, familiarize yourself with the following considerations:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="timing"
+    title="How long before I see my data in the New Relic UI?"
+   >
+   When you select your browser, mobile, and Lambda data sources, you should expect a delay of up to three minutes before you see your results in the New Relic UI for that trace observer. This allows the trace observer to wait long enough for these high-latency data sources to arrive before making a sampling decision . 
+
+   For example, if you re-run the [sample curl test](#sample-request) test after you add browser, mobile, or Lambda data sources, you should expect a delay before the data appears in the UI.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="multi-named-apps"
+    title="What if my browser app is connected to an APM agent that has multiple names?"
+  >
+    If your browser app is associated with an [APM application that has multiple names](/docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app/), you can only select one from the group of multi-named browser apps when you choose your data source. If you try to select the same application that goes by another name, the UI will prompt you that you need to choose only one browser app.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="lambda-sampling"
+    title="How does Lambda sampling work?"
+  >
+  By default, when you set up Lambda layers, you automatically get head-based sampling. When you enable Infinite Tracing for Lambda, your head-based sampling data is additionally sampled by the trace observer. The main benefit to enabling Infinite Tracing for your Lambda functions is to ensure their spans are included with spans from other services tracked by Infinite Tracing. 
+  </Collapser>
+</CollapserGroup>
+ 
+The instructions below are about using the New Relic UI, but if you prefer to take a programmatic approach, you can specify your trace observer data sources using [GraphQL](/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql/#updating-the-data-sources). When you finish, return to this page and continue with step 4 (Optional configurations) below. 
+
+To finish the configuration for browser, mobile, and Lambda in the UI: 
+
+1. Return to the trace observer Nerdlet (**[one.newrelic.com](https://one.newrelic.com) > Apps > Your Apps > New Relic Edge**).
```

Comment:
I would avoid using Nerdlet in this way, as it's generally just used in the context of creating New Relic One custom apps. I'd instead just refer to the 'New Relic Edge app.'.

Reply:
@zuluecho9, I went with "trace observer app."